### PR TITLE
release helm chart during travis release script

### PR DESCRIPTION
**Pull request by almerico**
part of Activiti/Activiti#3295

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +30,28 @@ before_install:
   - mkdir -p ${HOME}/.m2
   - cp settings.xml $HOME/.m2
 
+  - export PREVIEW_NAMESPACE=$(echo "v$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"|  tr '[:upper:]' '[:lower:]'|tr . -)
+  - ORG='activiti'
+  - export HELM_RELEASE_NAME="$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"
+  - export GLOBAL_GATEWAY_DOMAIN="35.242.205.159.nip.io"
+  - export GATEWAY_HOST="gateway.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN"
+  - export SSO_HOST="identity.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN"
+  - curl -Ls -o updatebot.jar   https://repo1.maven.org/maven2/io/jenkins/updatebot/updatebot/1.1.53/updatebot-1.1.53.jar
```
Comment (mteodori) [inline .travis.yml:39]: extract version in a UPDATEBOT_VERSION variable

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +30,28 @@ before_install:
   - mkdir -p ${HOME}/.m2
   - cp settings.xml $HOME/.m2
 
+  - export PREVIEW_NAMESPACE=$(echo "v$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"|  tr '[:upper:]' '[:lower:]'|tr . -)
+  - ORG='activiti'
+  - export HELM_RELEASE_NAME="$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"
+  - export GLOBAL_GATEWAY_DOMAIN="35.242.205.159.nip.io"
```
Comment (mteodori) [inline .travis.yml:36]: GLOBAL_GATEWAY_DOMAIN should be travis env variable

File: release-helm.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex
+
+git clone -b dev-almerico-AAE-2501-helm https://${GITHUB_TOKEN}:x-oauth-basic@github.com/Activiti/activiti-cloud-application.git
```
Comment (mteodori) [inline release-helm.sh:4]: we should change this branch name to develop

File: .travis.yml (hunk 1 above)
Comment (mteodori) [reply .travis.yml:39]: also JX_RELEASE_VERSION=1.0.24

File: .travis.yml (hunk 2 above)
Comment (almerico) [reply .travis.yml:36]: done

File: .travis.yml (hunk 1 above)
Comment (almerico) [reply .travis.yml:39]: done

File: release-helm.sh (hunk 3 above)
Comment (almerico) [reply release-helm.sh]: done

**Pull request merged**